livekit-rtc/livekit/rtc/participant.py

```diff
@@ -60,7 +60,7 @@ class Participant:
     def __init__(self, owned_info: proto_participant.OwnedParticipant) -> None:
         self._info = owned_info.info
         self._ffi_handle = FfiHandle(owned_info.handle.id)
-        self.tracks: dict[str, TrackPublication] = {}
+        self.trackPublications: dict[str, TrackPublication] = {}
```

Comment:
```suggestion
        self.track_publications: dict[str, TrackPublication] = {}
```

Reply:
realized this about 0.2 seconds after I pushed. fixed